src/collaboration-platform/GitHubCollaborationPlatform.ts

```diff
@@ -17,25 +41,126 @@ export class GitHubCollaborationPlatform implements CollaborationPlatform {
   }
 
   async getPRForCommit(commit: Commit): Promise<PullRequestInfo | null> {
-    // TODO: Implement
-    return null;
+    const owner = (await repoPathToOwnerAndRepo(this.repoPath)).owner;
+    const repo = (await repoPathToOwnerAndRepo(this.repoPath)).repo;
```

Comment:
nit: you can destructure the returned object instead of calling the function twice

```suggestion
    const { owner, repo } = await repoPathToOwnerAndRepo(this.repoPath);
```